src/Temporalio/Bridge/OptionsExtensions.cs

```diff
@@ -370,6 +374,9 @@ internal static class OptionsExtensions
                 max_task_queue_activities_per_second = options.MaxTaskQueueActivitiesPerSecond ?? 0,
                 graceful_shutdown_period_millis =
                     (ulong)options.GracefulShutdownTimeout.TotalMilliseconds,
+                use_worker_versioning = (byte)(options.UseWorkerVersioning ? 1 : 0),
+                // TODO: Expose to user
```

Comment:
@cretz FYI I left this for your discretion.

It might make sense to have the bridge layer actually allow nulls/optionals here, and that way you can rely on Core's defined defaults instead of having to re-assume them here.

Reply:
I think it's good for now